lint.sh

```diff
@@ -5,8 +5,43 @@ set -e
 FWDIR="$(cd "`dirname $0`"; pwd)"
 cd "$FWDIR"
 
+# https://stackoverflow.com/a/17841619
+function join {
+  local d=$1
+  shift
+  echo -n "$1"
+  shift
+  printf "%s" "${@/#/$d}"
+}
+
+include_dirs=(
+  "mlflow"
+  "tests"
+)
```

Comment:
We actually should include `examples` here, but that would complicate this PR. I'll address this in a separate follow-up.